src/azul/service/hca_response_v5.py

```diff
@@ -254,26 +241,12 @@ class KeywordSearchResponse(AbstractResponse, EntryFetcher):
     Not to be confused with the 'keywords' endpoint
     """
 
-    def _merge(self, dict_1: MutableJSON, dict_2: MutableJSON, identifier):
-        merged_dict: MutableMapping[str, List[AnyMutableJSON]] = defaultdict(list)
-        dict_id = dict_1.pop(identifier)
-        dict_2.pop(identifier)
-        for key, value in chain(dict_1.items(), dict_2.items()):
-            if isinstance(value, str):
-                merged_dict[key] = list(set(merged_dict[key] + [value]))
-            elif isinstance(value, int):
-                merged_dict[key] = list(merged_dict[key] + [value])
-            elif isinstance(value, list):
-                cleaned_list = list(filter(None, chain(value, merged_dict[key])))
-                if len(cleaned_list) > 0 and isinstance(cleaned_list[0], dict):
-                    # Make each dict hashable so we can deduplicate the list
-                    merged_dict[key] = list(map(thaw, set(map(freeze, cleaned_list))))
-                else:
-                    merged_dict[key] = list(set(cleaned_list))
-            elif value is None:
-                merged_dict[key] = []
-        merged_dict[identifier] = dict_id
-        return dict(merged_dict)
+    @classmethod
```

Comment:
AFAICT, this is only ever called through `self` and doesn't need to be a class method. In fact, since this is called so frequently, I'd consider making this a function in this module and calling it `_sort`.